continuous_integration/install.sh

```diff
@@ -33,6 +33,7 @@ conda install --yes numpy scipy matplotlib netcdf4 nose sphinx numpydoc hdf4=4.2
 conda install --yes sphinx_rtd_theme
 
 
+pip install git+https://github.com/jleinonen/pytmatrix.git
```

Comment:
I think we'll end up pinning it to a specific version of pytmatrix once Jussi updates the PyPI release, and then just updating the pin with new releases after we test?